src/main/java/com/breskul/bring/AnnotationConfigApplicationContext.java

```diff
@@ -222,10 +280,31 @@ private <T> Predicate<Map.Entry<String, Object>> filterByBeanType(Class<T> beanT
         return entry -> beanType.isAssignableFrom(entry.getValue().getClass());
     }
 
+    /**
+     * <h3>Finds bean from Application Context</h3>
+     * <p>finds bean if only one is expected</p>
+     *
+     * @param beanType {@link Class}
+     * @return {@link Object}
+     */
+
     @Override
     public <T> T getBean(Class<T> beanType) throws NoSuchBeanException, NoUniqueBeanException {
+        return getBean(beanType, true);
+    }
+
+    /**
+     * <h3>Finds bean from Application Context</h3>
+     * <p>Special form of getBean method if we expect to receive multiple beans</p>
+     *
+     * @param beanType        {@link Class}
+     * @param oneBeanExpected {@link Boolean}
+     * @return {@link Object}
+     */
+
+    public <T> T getBean(Class<T> beanType, boolean oneBeanExpected) throws NoSuchBeanException, NoUniqueBeanException {
```

Comment:
I think second param not necessary here.

Reply:
actually it is . pull the branch and doulbe check